pkgs/applications/audio/argotlunar/default.nix

```diff
@@ -0,0 +1,48 @@
+ { pkgs } :
+
+pkgs.stdenv.mkDerivation rec {
+  pname = "argotlunar";
+  version = "2.1.0";
+
+  src = pkgs.fetchgit {
+    url = "https://github.com/mourednik/argotlunar";
+    rev = "1527d5dd753a7af32600ac677337469bbf1d40cd";
+    sha256 = "sha256-U20ZUFrck/Y3GwqEnPZewnUN1YIFucgNsJJTMtWY3wE=";
+  };
+
+  buildInputs = [
+    pkgs.alsa-lib
+    pkgs.freetype
+    pkgs.curl
+    pkgs.pkg-config
+    pkgs.cmake
+    pkgs.xorg.libX11
+    pkgs.xorg.libXrandr
+    pkgs.xorg.libXinerama
+    pkgs.xorg.libxshmfence
+    pkgs.xorg.libXext
+    pkgs.xorg.libXcursor
+  ];
+
+  configurePhase = ''
```

Comment:
I'm surprised that you need this and the `buildPhase` -- having `cmake` as part of your deps should make everything work automagically. Can you clarify what doesn't work with the standard configurePhase/buildPhase?

Reply:
This might not be necessary. I was following documentation online (I think it was [this guide](https://nix-tutorial.gitlabpages.inria.fr/nix-tutorial/first-package.html)) and copied the example. Let me give it a try without this phase.

Reply:
I tried removing the configure and the build phase (both and either one), and the build would fail. It seems like some paths are hardcoded in the upstream repo:

https://github.com/mourednik/argotlunar/blob/1527d5dd753a7af32600ac677337469bbf1d40cd/JUCE/tools/Makefile#L60

And without specifying `cmake-build` it fails because it looks for file in the non-existing directory.